### PR TITLE
now draft message tells which deployment is ready to go

### PR DESCRIPTION
**Pull request by xiujiao**

### CODE REVIEW

**Comment by bodymindarts** [comment]
Its working in our env

**Comment by xiujiao** [comment]
@bodymindarts Thank you :)

File: bin/genesis [hunk 1]
```diff
@@ -4570,9 +4570,8 @@ EOF
 
 cmd_ci_draft_message() {
 	[[ -z $1 ]] || bad_usage "ci draft-message"
-
 	(
-		echo "[${CI_SITE_ENV}] Deployment changes are ready to go!"
+		awk '/^name: /{print "[" $2 "] Deployment changes are ready to go!"}' "${LOCAL_CHANGES}/${CI_SITE_ENV}/name.yml"
```
Comment (jhunt) [inline bin/genesis]: What happens if `$.name` is a Spruce operator?

Comment (xiujiao) [inline bin/genesis]: @jhunt Do you mean in `name.yml`, the name is specified by Spruce operator? 
In the current `genesis` version, name is got  by`name: ${site}-${name}-${DEPLOYMENT_NAME}, so in the `name.yml`, I can retrieve the name correctly without worrying about the case it is specified by Spruce operator. If in future, it does use Spruce operator to specify name, I can get name from `manifest.yml`, but the downside is that we have to generate `manifest.yml` for prod and push it before we configure the pipeline.
Any other good idea about this?

**Comment by geofffranks** [comment]
Closing, since genesis-v2 obviates this

**Pull request closed without merging**